src/raylib/textures/Image.hpp

```diff
@@ -0,0 +1,147 @@
+/*
+** EPITECH PROJECT, 2022
+** Bomberman-Global-Offensive
+** File description:
+** Image
+*/
+
+#ifndef IMAGE_HPP_
+#define IMAGE_HPP_
+
+extern "C"
+{
+#include "raylib.h"
+}
+
+namespace raylib
+{
+    namespace textures
+    {
+        class Image {
+          public:
+            /// This function is a constructor for the Image class. It takes in a
+            /// Vector2 position, a Color pointer, and a char pointer. It sets the
+            /// position to the position passed in, sets the fileName to the fileName
+            /// passed in, and then calls the loadImage function
+            ///
+            /// @param position The position of the image on the screen.
+            /// @param color The color of the image.
+            /// @param fileName The name of the file to load.
+            Image(Vector2 position = {}, Color *color = {}, char *fileName);
+
+            /// Destructor of Image class
+            ~Image() = default;
+
+            /// LoadImage loads an image from a file
+            ///
+            /// @param fileName The name of the file to load.
+            void loadImage(const char *fileName);
+
+            /// Loads an image from a texture
+            ///
+            /// @param texture The texture to load the image from.
+            void loadImageFromTexture(Texture2D texture);
+
+            /// It loads an image from the screen
+            void loadImageFromScreen(void);
+
+            /// UnloadImage() unloads the image from memory
+            void unloadImage();
+
+            /// It exports the image to a file.
+            ///
+            /// @param fileName The name of the file to export the image to.
+            ///
+            /// @return A boolean value.
+            bool exportImage(const char *fileName);
+
+            /// Returns a new image that is a sub-image of the current image
+            ///
+            /// @param rec The rectangle to get the image from.
+            ///
+            /// @return A new image object.
+            Image imageFromImage(Rectangle rec);
+
+            /// It changes the format of the image.
+            ///
+            /// @param newFormat The new format for the image.
+            void imageFormat(int newFormat);
```

Comment:
What is the format parameter ? Is there more precision in the raylib documentation ?